### PR TITLE
remove log 5.5 and deprecate log 5.6

### PR DESCRIPTION
**Pull request by anpingli**
By the way, unless, customer request maintenance support . I will remove  the deprecated version after 6 months.

### CODE REVIEW

**Comment by anpingli** [comment]
@QiaolingTang  @kabirbhartiRH  PTAL

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-openshift-logging-5.6.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ resources:
       memory: 4Gi
 tests:
 - as: azure-ipi-fips-f60
```
Comment (QiaolingTang) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-openshift-logging-5.6.yaml:54]:
```suggestion
- as: azure-ipi-fips-f365
```

And same comment to other files.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-openshift-logging-5.6.yaml [hunk 2]
```diff
@@ -51,8 +51,8 @@ resources:
       cpu: "1"
       memory: 4Gi
 tests:
-- as: azure-ipi-fips-f60
-  cron: 7 4 12 2,4,6,8,10,12 *
+- as: azure-ipi-fips-f365
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-openshift-logging-5.6.yaml:54]: I think we are using `f360` for the other jobs.

**Comment by anpingli** [comment]
update the PR. Add back 5.9 as 5.9 extend extend the support for that release until 6.2.

**Comment by anpingli** [comment]
@liangxia  PTAL

**Pull request merged**